predicthq/version.py

```diff
@@ -1 +1 @@
-__version__ = "1.9.4"
+__version__ = "1.10.0"
```

Comment:
Wasn't too sure if I should bump the version here in my branch or after the merge to master? Also the tag should be created after the merge to master right?